src/scripts/refgene2protein.pl

```diff
@@ -333,11 +356,19 @@ sub parse_one_genbank {
   my $desc = $record->desc() || die "no desc for $accession";
 
   my $cds_count = 0;
-  my $transcript_variant = $desc =~ /transcript variant (\d+)/ ? $1 : 0;
+#  my $transcript_variant = $desc =~ /transcript variant (\d+)/ ? $1 : 0;
+#  my $transcript_variant = $desc =~ /transcript variant ([\w\-]+)/ ? $1 : 0;
+  my $transcript_variant = $desc =~ /transcript variant ([^,]+)/ ? $1 : "";
```

Comment:
This will match up to the end of $desc, is that's what's desired, or do you only want the next word?

Reply:
So I believe these lines are always formatted to have a comma after the transcript variant label, the regexp is actually matching "any run of characters except a comma".  I went through a few iterations of this mentioned in the nearby comments, there are sometimes spaces as well, etc. so this route seemed the most flexible.